OracleDatabase/OracleConnectionManager/dockerfile/12.2.0.1/checkSpace.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# LICENSE UPL 1.0
+#
+# Copyright (c) 1982-2018 Oracle and/or its affiliates. All rights reserved.
+#
+# Since: January, 2018
+# Author: paramdeep.saini@oracle.com
+# Description: Checks the available space of the system.
+# 
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+# 
+
+REQUIRED_SPACE_GB=6
+AVAILABLE_SPACE_GB=`df -B 1G / | tail -n 1 | awk '{print $4}'`
```

Comment:
Please update this to match the updated single instance test, i.e. `df -PB 1G /`